vue-components/src/components/ToggleButtonGroup.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<span class="wikit wikit-ToggleButtonGroup">
+		<slot />
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export interface ToggleButtonGroupInjection {
+	groupValue: ( () => string ) | null;
+	toggleListener: ( ( value: string ) => void ) | null;
+}
+
+export default Vue.extend( {
+	name: 'ToggleButtonGroup',
+	provide(): ToggleButtonGroupInjection {
+		return {
+			groupValue: (): string => this.value,
+			toggleListener: ( event: string ): void => {
+				this.$emit( 'input', event );
+			},
+		};
+	},
+	props: {
+		value: {
+			required: true,
+			type: String,
+		},
+	},
+} );
+</script>
+<style lang="scss">
+.wikit-ToggleButtonGroup :not(:last-child) {
+	border-inline-end: none;
+	border-start-end-radius: 0;
+	border-end-end-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child) {
+	border-start-start-radius: 0;
+	border-end-start-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child) {
+	border-start-start-radius: 0;
+	border-end-start-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :disabled {
+	border-block: none;
+}
+
+.wikit-ToggleButtonGroup :focus {
+	outline: none;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child):disabled {
+	border-inline-start-color: $color-base-100;
+}
```

Comment:
Please follow the pattern established in other components and either use a variable for the base class name or nest all these rules under the base class.

Reply:
ahaaaa I did `& :not(:last-child)` and got stylelint error `35:3  ✖  Unnecessary nesting       scss/selector-no-redundant-nesting-selector` now I realize what was wrong. Thanks.